packages/select/src/ResourceSelect.js

```diff
@@ -192,15 +214,23 @@ const ResourceSelect = ({
     }
     return fetch()
       .then(async resp => {
-        if (!resp || !resp.data) {
+        if (
+          (!shouldFilterByAllOptions && !resp?.data) ||
+          (shouldFilterByAllOptions && !resp)
+        ) {
           throw new Error(`API returned an invalid response.`);
         }
         const getResult = rest.getResult || resource.getResult;
 
-        const items =
-          typeof getResult === 'function'
-            ? getResult.call(resource, resp.data)
-            : resp.data[getResult];
+        let items = [];
+        if (shouldFilterByAllOptions) {
+          items = resp;
+        } else {
+          items =
+            typeof getResult === 'function'
+              ? getResult.call(resource, resp?.data)
```

Comment:
this would be a breaking change for people using this package that haven't upgraded to the latest version of workflow. we're not using optional chaining anywhere else in `availity-react`. i don't think we're ready to require people to upgrade to use this package.